apps/studio/components/interfaces/TableGridEditor/SidePanelEditor/ColumnEditor/ColumnEditor.tsx

```diff
@@ -94,22 +93,25 @@ const ColumnEditor = ({
   })
   const foreignKeyMeta = data || []
 
-  const keys = vault.listKeys()
+  const { data: keys } = usePgSodiumKeysQuery({
```

Comment:
actually i dont think we need this anymore as we're no longer supporting TLE on the GUI